doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -134,6 +138,215 @@ Examine the FluentBit sidecar's STDOUT to see logs for Ray's component processes
 kubectl logs raycluster-complete-logs-head-xxxxx -c fluentbit
 ```
 
+(kuberay-fluentbit-daemonset)=
+## Setting up logging DaemonSet with Fluent Bit
+
+In this section, we give a walkthrough of how to set up log-emitting [Fluent Bit][FluentBit] DaemonSet for Ray pods.
+The following tutorial will be based on AWS EKS, but the same steps can be applied to other Kubernetes distributions.
+
+### Setup for AWS EKS
+
+Create an EKS cluster named `fluent-bit-demo`, as shown in the [EKS getting started guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+
+Create a policy file called `eks-fluent-bit-daemonset-policy.json` with the following content:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "firehose:PutRecordBatch"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "logs:PutLogEvents",
+            "Resource": "arn:aws:logs:*:*:log-group:*:*:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogStream",
+                "logs:DescribeLogStreams",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "arn:aws:logs:*:*:log-group:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "logs:CreateLogGroup",
+            "Resource": "*"
+        }
+    ]
+}
+```
```

Comment:
The policy file is a bit opaque for beginners, could we add a brief description of what the different parts do?